superset-frontend/.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "singleQuote": true,
-  "trailingComma": "all"
+  "trailingComma": "all",
+  "arrowParens": "avoid"
```

Comment:
`prettier` changed the default behavior for `arrowParent`. We revert it back to the old behavior.

Reply:
There seems to be another change to space between `function` and parentheses.

`function() {}` vs `function () {}`

I do not have strong preference in terms of styling but perhaps we can use the original to reduce amount of changes. 

This is the settings from `@superset-ui`

```ts
{
  "arrowParens": "avoid",
  "bracketSpacing": true,
  "jsxBracketSameLine": false,
  "printWidth": 100,
  "proseWrap": "always",
  "requirePragma": false,
  "semi": true,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
  "useTabs": false
};
```

Reply:
https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot

It seems Prettier folks decided this is somehow the preferred style and didn't add an option to turn it off.

Reply:
ic. If no strong objection from others I am good with this. I'll approve but wait for others to see a bit.

Reply:
I prefer without the space, but if there's no way to turn it off, it won't break my heart.

Reply:
I just read through some of the threads on that feature, and now I'm glad I don't work on that project. Yeesh!

Reply:
Can we merge this then? 😄